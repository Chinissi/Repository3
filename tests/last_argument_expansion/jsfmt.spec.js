run_spec(__dirname, ["flow"]);
run_spec(__dirname, ["flow"], { jsxBracketSpacing: true });
