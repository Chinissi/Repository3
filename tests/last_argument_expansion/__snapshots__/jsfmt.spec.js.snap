exports[`test jsx.js 1`] = `
"const els = items.map(item => (
  <div className=\"whatever\">
    <span>{children}</span>
  </div>
));
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
const els = items.map(item => (
  <div className=\"whatever\">
    <span>{children}</span>
  </div>
));
"
`;

exports[`test object.js 1`] = `
"const formatData = pipe(
  zip,
  map(([ ref, data ]) => ({
    nodeId: ref.nodeId.toString(),
    ...attributeFromDataValue(ref.attributeId, data)
  })),
  groupBy(prop(\'nodeId\')),
  map(mergeAll),
  values
);

export const setProp = y => ({
  ...y,
  a: \'very, very, very long very, very long text\'
});

export const log = y => {
  console.log(\'very, very, very long very, very long text\')
};
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
const formatData = pipe(
  zip,
  map(([ref, data]) => ({
    nodeId: ref.nodeId.toString(),
    ...attributeFromDataValue(ref.attributeId, data)
  })),
  groupBy(prop(\"nodeId\")),
  map(mergeAll),
  values
);

export const setProp = y => ({
  ...y,
  a: \"very, very, very long very, very long text\"
});

export const log = y => {
  console.log(\"very, very, very long very, very long text\");
};
"
`;
